packages/iTwinUI-react/src/core/Slider/Slider.test.tsx

```diff
@@ -28,6 +28,7 @@ const createBoundingClientRect = (
  */
 const getBoundingClientRect = Element.prototype.getBoundingClientRect;
 const sliderContainerSize = createBoundingClientRect(10, 0, 1010, 60);
+const sliderContainerVerticalSize = createBoundingClientRect(10, 0, 70, 1000);
 Element.prototype.getBoundingClientRect = () => sliderContainerSize;
```

Comment:
I would recommend to change this block (29-36) to something like this:
```ts
const getBoundingClientRectMock = jest.spyOn(HTMLElement.prototype, 'getBoundingClientRect');
const sliderContainerHorizontalSize = createBoundingClientRect(10, 0, 1010, 60);
const sliderContainerVerticalSize = createBoundingClientRect(10, 0, 70, 1000);

beforeEach(() => {
  getBoundingClientRectMock.mockReturnValue(sliderContainerHorizontalSize);
});

afterAll(() => {
  jest.clearAllMocks();
});
```
And then inside vertical tests you need change mock to:
```ts
getBoundingClientRectMock.mockReturnValue(sliderContainerVerticalSize);
```

Reply:
I replaced the lines as you suggested. To confirm, would it be alright to add `getBoundingClientRectMock.mockReturnValue(sliderContainerVerticalSize);` to only the tests that depend on the dimension of the slider container? As of now, I only added that line to the last five vertical tests since they are the only vertical tests that fail with this line (depend on the dimension of the slider container)